src/QuickMarcEditor/SourceFileLookup/SourceFileLookupModal.test.js

```diff
@@ -58,15 +58,15 @@ describe('Given SourceFileLookupModal', () => {
   describe('when some source file value is selected', () => {
     it('should enable Save & close button', async () => {
       const {
-        getByRole,
         getByLabelText,
+        getByText,
       } = renderSourceFileLookupModal();
 
       const select = getByLabelText('ui-quick-marc.sourceFileLookupModal.fieldLabel');
 
       fireEvent.change(select, { target: { value: sourceFileOptions[0].value } });
 
-      const button = getByRole('button', { name: 'stripes-components.saveAndClose' });
+      const button = getByText('stripes-components.saveAndClose').closest('button');

```

Comment:
Was there an issue with `getByRole`?

Reply:
`Modal` component elements are no longer accessible via `getByRole` due to a recent refactoring.